docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -538,6 +538,78 @@ as follows:
 Determining Whether A Simulant is Affected - Iron
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+**Hemoglobin Level**
+
+[This section in another PR that I will have to fix a merge conflict for]
+
+**Birth Weight**
+
+Our model will apply the effect size of maternal consumption of iron fortified 
+foods on infant birth weight under the following assumptions:
+
+1. The effect of consuming iron fortified foods takes place six months after 
```

Comment:
What was the effect measured by this paper? Was it hemoglobin level, or did they also look at birth weight?

If it's the former, do we actually need to take this delay into account for the effect on birth weight? Why can't we just use whatever timeframe is in the studies from the Haider paper, since those are directly measuring the effect we are interested in?

Reply:
Also note that we did **not** incorporate a similar delay for folic acid. We just said that as long as the mother *starts* eating fortified food at least 3 months before she gets pregnant, her baby will have a lower risk of neural tube defects.

Unless we have data to tell us otherwise, it seems like we should be consistent in our strategies: Either update the folic acid intervention to incorporate extra time for the fortification to take effect; or for the iron intervention, assume that as long as the mother starts getting the iron by the recommended time (e.g. 3 months before birth), it will have an effect on birth weight.

Reply:
Hmm. The Andersson paper looked at hemoglobin response in adult women taking iron supplementation and showed that there is a scale up to peak response at approximately 6 months - they did not evaluate birth weight at all. 

My thinking behind this is that the causal pathway is likely [iron supplementation/fortification] -> [increased maternal hemoglobin] -> [increased infant birth weight] (an intermediate through maternal hemoglobin rather than directly between iron supp. and birth weight). The Haider paper presents that maternal hemoglobin in the third trimester is significantly associated with birth weight, which supports this claim. Especially because our intervention is fortification and we are using supplementation literature here, I was thinking that including the 6 month lag time ensures that there will actually be a response in maternal hemoglobin following fortification which will then in turn affect birth weight. I think it is definitely possible that including both the 6 month maternal response time as well as the 3 month pregnancy lag is overly conservative, though - it was my intention to be conservative, so we could consider amending this (maternal hemoglobin at delivery was also associated with birth weight; although I don't think that they controlled for maternal hb at 6 months gestation... I'm happy to look into this more if desired!)

Reply:
Also, it seems like the 3 months before pregnancy is inherently a sort of scale-up period? Because the folate levels *at conception* is the important part, right? I'm not really familiar with the topic, but that was my understanding.

Reply:
I'll pull some of the individual studies in the Haider paper and see if those provide any insight.

Reply:
I have added a to-do note in this current PR to investigate this issue and update the document as necessary. I am working on investigating in the meantime.

Reply:
Seems reasonable.  The current strategy is a conservative approach, so that's probably a good default.  We should just be sure to carefully document our decisions for both iron and folic acid, citing the appropriate references.

Reply:
"Regulation of iron transfer to the fetus
Transfer of iron from the mother to the fetus is supported by
a substantial increase in maternal iron absorption during
pregnancy and is regulated by the placenta (3, 4). Serum ferritin
usually falls markedly between 12 and 25 wk of gestation, probably
as a result of iron utilization for expansion of the maternal
red blood cell mass. Most iron transfer to the fetus occurs after
week 30 of gestation, which corresponds to the time of peak
efficiency of maternal iron absorption. Serum transferrin carries
iron from the maternal circulation to transferrin receptors
located on the apical surface of the placental syncytiotrophoblast,
holotransferrin is endocytosed, iron is released, and
apotransferrin is returned to the maternal circulation. The free
iron then binds to ferritin in placental cells where it is transferred
to apotransferrin, which enters from the fetal side of the
placenta and exits as holotransferrin into the fetal circulation.
This placental iron transfer system regulates iron transport to
the fetus. When maternal iron status is poor, the number of placental
transferrin receptors increases so that more iron is taken
up by the placenta. Excessive iron transport to the fetus may be
prevented by the placental synthesis of ferritin. As discussed
later in this review, evidence is accumulating that the capacity
of this system may be inadequate to maintain iron transfer to the
fetus when the mother is iron deficient."

Reply:
have we looked into this? 

The relation between maternal anemia and birth weight has
been reviewed more extensively elsewhere in this issue (15). In
several studies, a U-shaped association was observed between
maternal hemoglobin concentrations and birth weight (16).
Abnormally high hemoglobin concentrations usually indicate
poor plasma volume expansion, which is also a risk for low birth
weight (15, 17). Lower birth weights in anemic women have
been reported in several studies (18–20). In a multivariate regression
analysis of data from 691 women in rural Nepal, adjusted decrements in neonatal weight of 38, 91, 187, and 153 g were
associated with hemoglobin concentrations ³20, 90–109, 70–89
and < 70 g/L, respectively. The odds for low birth weight were
increased across the range of anemia, increasing with lower
hemoglobin in an approximately dose-related manner (1.69,
2.75, and 3.56 for hemoglobin concentrations of 90–109, 70–89,
and 110–119 g/L, respectively) (21). Trials that included large
numbers of iron-deficient women showed that iron supplementation
improved birth weight (19, 22).
Some investigators reported a negative association between
maternal serum ferritin and birth weight and a positive association
with preterm delivery (23–25). These findings probably
indicate the presence of infection, which elevates serum ferritin. 



-Dreyfuss M. Anemia and iron deficiency during pregnancy: etiologies
and effects on birth outcomes in Nepal. PhD dissertation. Johns
Hopkins University, Baltimore, 1998.

Reply:
It seems like the Allen paper supports our assumptions here in that they say most iron transfer from mother to fetus occurs after week 30 of gestation (aka in the third trimester). And since we are calculating our impact based on the fortification dose, I think that it is appropriate to include the time-to-response period for the fortification dose. We could potentially consider a scale-up or overlap of these time periods (maybe it doesn't have to be 6 months + 3 months, but rather maximum of 6 months and 3 months, something in between that, etc.). I said I will look into the individual studies included in the Haider paper and I've started to do that and will bump it up to higher priority!

Reply:
So still looking into the original papers, but upon a closer read of the Haider meta-analysis, I realized that they included studies with duration as low as 7 weeks (max 30 weeks) and reported that their was no evidence that duration of supplementation influenced the effect on birth weight after adjusting for dose. So we could use 7 weeks as the lower bound for time-to-response. Thoughts?

Reply:
it seems like between 7-36 weeks is when plasma volume expansion occurs and where adequate iron stores is required! (so could be our lower bound) Too much iron is bad for birthweight too. It is a U shaped curve. Do we want to model this effect? If we supplement everyone, there is a chance that ppl who already have high hb concentrations iwll get too much iron stores and have lbw babies? 

In normal pregnancies, plasma volume expansion leads to a gradual reduction in haemoglobin concentration and haematocrit levels in the gestational age period between 7 and 36 weeks, despite concurrent increases in the total erythrocyte and haemoglobin mass [5]. This haemodilution is believed to increase oxygen delivery to tissues and organs by reducing blood viscosity and thus increasing blood velocity, particularly in the microcirculatory vessels of the maternal intravillous space. Thus, there is an optimum haematocrit level for oxygen delivery that is lower than haematocrit values found in healthy, non-pregnant women [6]. Haemodilution is vital for a healthy pregnancy, and analyses of large numbers of women of various races have shown that there is a pronounced U-shaped curve with the lowest risk of low birth weight and prematurity occurring well below the WHO definition of anaemia [7].

In iron-deficient women, haemodilution without an adequate supply of iron to the developing erythrocyte and haemoglobin mass will likely result in placental insufficiency, leading to low birth weight and prematurity. This may explain the negative association that we found between iron status and birthweight in women who were not supplemented with iron (Fig. 1, red line). Conversely, when iron deficiency is reversed by iron supplementation, birth weight is increased. With increasing iron status, however, the elevated haematocrit levels due to iron supplementation will increase blood viscosity. At some point, haematocrit levels will exceed the optimum value, resulting in a decreased delivery of oxygen (and other nutrients) and decreasing birth weight (Fig. 1, blue line). The crossing of red and blue lines in our figure suggests that iron supplementation can even lead to a reduced birth weight in women with high initial iron status, but this may have affected very few women (grey line) and supportive evidence is weak (see 95% confidence bands in right panel).

Reply:
Right, I think because fortification is such a small amount compared to supplementation (like 3 mg vs. 60+ mg) we won't have to consider it. Maybe we could state that we assume those who are taking iron supplementation during pregnancy will consider the extra dose from fortification and will adjust their supplementation dose accordingly.

Reply:
Also, Nicole, I'm not quite sure how to interpret the 7-36 week range for our purposes. Can you expand upon this if you think it will be of use to us?

Reply:
Do you think there is a risk for excess iron from supplemented food?

Reply:
t seems like between 7-36 weeks is when plasma volume expansion occurs and where adequate iron stores is required! Too much iron is bad for birthweight too. It is a U shaped curve. Do we want to model this effect? If we supplement everyone, there is a chance that ppl who already have high iron levels concentrations will get too much iron stores and have lbw babies?

"In normal pregnancies, plasma volume expansion leads to a gradual reduction in haemoglobin concentration and haematocrit levels in the gestational age period between 7 and 36 weeks, despite concurrent increases in the total erythrocyte and haemoglobin mass [5]. This haemodilution is believed to increase oxygen delivery to tissues and organs by reducing blood viscosity and thus increasing blood velocity, particularly in the microcirculatory vessels of the maternal intravillous space. Thus, there is an optimum haematocrit level for oxygen delivery that is lower than haematocrit values found in healthy, non-pregnant women [6]. Haemodilution is vital for a healthy pregnancy, and analyses of large numbers of women of various races have shown that there is a pronounced U-shaped curve with the lowest risk of low birth weight and prematurity occurring well below the WHO definition of anaemia [7].

In iron-deficient women, haemodilution without an adequate supply of iron to the developing erythrocyte and haemoglobin mass will likely result in placental insufficiency, leading to low birth weight and prematurity. This may explain the negative association that we found between iron status and birthweight in women who were not supplemented with iron (Fig. 1, red line). Conversely, when iron deficiency is reversed by iron supplementation, birth weight is increased. With increasing iron status, however, the elevated haematocrit levels due to iron supplementation will increase blood viscosity. At some point, haematocrit levels will exceed the optimum value, resulting in a decreased delivery of oxygen (and other nutrients) and decreasing birth weight (Fig. 1, blue line). The crossing of red and blue lines in our figure suggests that iron supplementation can even lead to a reduced birth weight in women with high initial iron status, but this may have affected very few women (grey line) and supportive evidence is weak (see 95% confidence bands in right panel)."

-https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-019-1375-9

Do you think there is risk for excess iron from supplemented foods?

Reply:
Ya, I agree to use a shorter period for time to response.

Reply:
I think we are currently saying the effect of iron needs to kick by third trimester, and it takes 6 months before it starts working, so the mother needs to be eating fortified food at start of pregnancy (9 months before birth) for there to be an effect on birth weight? 

I was thinking that the effect should kick in by week 7 give or take (month 1.5 lets say). And if we want to say time to response is nil, then it just means the baby needs to be born to mother who started iron 7.5 months before birth for there to be ab effect on birth weight?

did I understand it correctly?

Reply:
sorry, my comments are showing up double :(

Reply:
Ah, ok, so you think that the mother needs to start consuming iron fortified foods at least by week 7 of gestation aka 7.5 months before birth because of the 7-36 week figure cited from the Whittaker 1996 paper?

I was suggesting that we use 7-30 weeks before birth as our range because that is the range of supplementation duration observed in the Haider paper which we use for our effect size and they reported that they did not observe any effect of duration of supplementation on birth weight after they corrected for dose, which implies that any duration in that range should be effective. To maximize impact, as Abie has been suggesting because the overall conclusion of the project will likely be that impact is *small*, I wonder if we can just move forward with a 7 week point estimate. What do you think of that?

Reply:
We could try to re-do the 2013 meta-analysis to include the more recent individual studies you have been finding, but that seems outside of the scope of this project!

Reply:
Ok, there is an excel sheet that summarizes gestational age at start of supplementation and duration of supplementation in the studies used in the Haider paper in the google drive folder for the Haider studies. It looks like on average, the studies start supplementation around 20 weeks of gestation aka approximately 20 weeks (~5 months) prior to birth assuming a full term pregnancy. Maybe we can go with this figure? Thoughts?

Reply:
Ok! I think this sounds good as we have a source to reference. At some point in the future, if we want to publish this, as should do a comprehensive review as Ali suggested. We could also run our assumptions by a nutritionist in the future.

Reply:
Agreed!!

Reply:
I just thought of something - we're assuming all births are exactly 40 weeks, but we actually *have* gestational ages built into our model. So if we wanted to be more accurate, could we use the simulant's gestational age to help determine whether they should get the birthweight effect from iron?